### PR TITLE
complete zanko

### PR DESCRIPTION
**Pull request by zankoav**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
             >
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#">Link 101</a>
+            <a class="nav-link" href="#">Link 41241</a>
```
Comment (alzan) [inline index.html]: must be 412

**Review by alzan — CHANGES_REQUESTED** [review]
Important from alzan

File: index.html (hunk 1 above)
Comment (alzan) [reply index.html]: I resolve it

File: index.html [hunk 2]
```diff
@@ -41,7 +41,7 @@
             >
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#">Link 41241</a>
+            <a class="nav-link" href="#">Link 412</a>
```
Comment (alzan) [inline index.html]: now need 40

**Review by alzan — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (zankoav) [reply index.html]: done

**Review by alzan — APPROVED** [review]

**Pull request merged**